vaadinfrontend/frontend/themes/datamanager/components/vaadin-button.css

```diff
@@ -0,0 +1,20 @@
+:host(.dialogue-button) {
+  width: 9rem;
+  height: 9rem;
+  border-radius: 1rem;
+  padding: 0;
+}
+
+:host(.dialogue-button) ::slotted(vaadin-icon){
+  width: 1.5rem;
+  height: 1.5rem;
+  padding: 0;
+}
+
+:host(.dialogue-button) [part="label"]{
+  font-size: var(--lumo-font-size-s);
+}
+
+:host(.dialogue-button) .vaadin-button-container {
+  display: grid;
+}
```

Comment:
This is unnecessary since the text is not within the button and so there is no space conflict with the icon. 
```suggestion
```